.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Auto detect text files and perform LF normalization
+* text=auto
+package.json text eol=lf 
```

Comment:
Is the `.gitattributes` file needed? Are you sure it's not added just because of EOL errors?

Reply:
This is so that someone publishing to npm doesn't accidentally push a CRLF version of `bin/obj2gltf.js`, which could happen if they are on Windows and using the default git settings.

Reply:
In that case, this would only enforce LF on package.json right? It would put `bin/obj2gltf` under the `* text=auto` rule.

https://help.github.com/articles/dealing-with-line-endings/#example

Reply:
Heh yeah my reason was totally wrong.

I think this was the actual reason: https://github.com/Microsoft/vscode-python/issues/355. Not sure if this is still an issue... I'll check.